source/examples/rapids-sagemaker-higgs/notebook.ipynb

```diff
@@ -26,8 +26,10 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    !docker build -t {estimator_info['ecr_image']} --build-arg RAPIDS_IMAGE={{ rapids_container }} .
</pre><p>This was fully broken. <code>docker build --build-arg</code> expects args in the form <code>{NAME}={VALUE}</code> . That space in between led to the build failing like this:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">"docker build" requires exactly 1 argument.
See 'docker build --help'.


Usage:  docker build [OPTIONS] PATH | URL | -


Build an image from a Dockerfile
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>